src/app.module.ts

```diff
@@ -10,20 +10,23 @@ import { VerifySignatureMiddleware } from './common/http-signature/verify-signat
 import { InboxController } from './inbox/inbox.controller';
 
 export const AppModuleConfig = {
-  imports: [
-    ConfigModule,
-    RabbitMQModule,
-    QueueModule,
-    DispatcherModule,
-    InboxModule
-  ],
+  imports: [ConfigModule, RabbitMQModule, QueueModule, DispatcherModule, InboxModule],
   controllers: [AppController],
   providers: [AppService],
 };
 
+// @Module(AppModuleConfig)
+// export class AppModule {
+//   configure(consumer: MiddlewareConsumer) {
+//     consumer.apply(VerifySignatureMiddleware).forRoutes(InboxController);
+//   }
+// }
+
 @Module(AppModuleConfig)
 export class AppModule {
   configure(consumer: MiddlewareConsumer) {
-    consumer.apply(VerifySignatureMiddleware).forRoutes(InboxController);
+    consumer
+      .apply(VerifySignatureMiddleware)
+      .forRoutes({ path: 'inboxes/:address/:hash', method: RequestMethod.DELETE });
```

Comment:
Signature verification should happen for all methods of the inbox controller, not only the delete method.